bower.json

```diff
@@ -1,17 +1,15 @@
 {
-  "name": "convert-ember-components",
+  "name": "lytics-ember-components",
```

Comment:
Oops. I'm sure this was my fault :blush: Kinda weird that bower assumed the branch name as the package name.